File: app/src/main/java/com/example/sunkai/heritage/Data/classifyActiviyData.java
package com.example.sunkai.heritage.Data;

import java.io.Serializable;

/**
 * Created by sunkai on 2017/3/27.
 * 此类用于存储首页5个页面的页面信息
 * 实现了Serializable可以传入至bundle中
 */

public class classifyActiviyData implements Serializable{
    public int id;
    public String activityTitle,activitContent,activityChannel;
    public byte[] activityImage;
}

File: app/src/main/java/com/example/sunkai/heritage/Data/classifyActiviyData.kt
package com.example.sunkai.heritage.Data

import java.io.Serializable

/**
 * Created by sunkai on 2017/12/15.
 * 首页非遗分类的listview显示数据
 */
class classifyActiviyData(var id:Int=0
                          ,var activityTitle:String?=null
                          ,var activityContent:String?=null
                          ,var activityChannel:String?=null
                          ,var activityImage:ByteArray?=null) : Serializable
